# Welcome to Galaxy-P Metaproteomics Website

<div style="text-align: justify"> Galaxy-P platform offers an open source, bioinformatics platform for disseminating metaproteomics software and workflows. This was a result of Metaproteomics contribution-fest(z.umn.edu/mphack2016) which generated easily accessible and documented metaproteomic software tools and workflows. The metaproteomic workflows encompass the core requirements i.e. a) Database generation, b) Peptide spectral matching, c) Taxonomic analysis and d) Functional analysis. These workflows can be used for training researchers and also broaden the use of the developed tools.</div> 

### Accessible links to the Metaproteomics training material. 


* [**_Instructions on using this gateway_**](http://z.umn.edu/SuppS1){:target="_blank"}

* [**_Access to Paper_**](http://www.mdpi.com/2227-7382/6/1/7/html){:target="_blank"}

* [**_Metaproteomics GTN_**](http://galaxyproject.github.io/training-material/topics/proteomics/){:target="_blank"}

* [**_Galaxy toolshed_**](https://toolshed.g2.bx.psu.edu/){:target="_blank"}

* [**_Galaxy Proteomics Github_**](https://github.com/galaxyproteomics){:target="_blank"}

* [**_Galaxy-P Metaproteomics Docker_**](https://github.com/galaxyproteomics/docker-galaxyp){:target="_blank"}

* [**_Metaproteomics Tutorial Video_**](http://z.umn.edu/mpvideo2018){:target="_blank"}


  
<br />

The [**_Galaxy-P team_**](https://galaxyp.org){:target="_blank"} collaborates with **Galaxy developers** around the world including the **European Galaxy Community**. Users are recommended to use [**_usegalaxy.eu_**](https://usegalaxy.eu){:target="_blank"} as an alternative server to run your analyses.

